pkg/stacks/constants.go

```diff
@@ -25,7 +31,9 @@ const (
 type AnchorMode uint8
 
 const (
-	AnchorModeOnChainOnly AnchorMode = 0x01
+	AnchorModeOnChainOnly  AnchorMode = 0x01
+	AnchorModeOffChainOnly AnchorMode = 0x02
+	AnchorModeAny          AnchorMode = 0x03
```

Comment:
I think only AnchorModeOnChainOnly is being used currently. Similar to payload type, I did not want to include unused anchor modes.

Reply:
Anchor mode is indeed deprecated in the latest stacks.js, however, the default one was AnchorModeAny.

Reply:
https://github.com/hirosystems/stacks.js/blob/main/packages/transactions/src/transaction.ts#L72
https://github.com/hirosystems/stacks.js/blob/main/packages/transactions/src/transaction.ts#L109